Sources/JXKit/JXErrors.swift

```diff
@@ -8,27 +8,31 @@ public enum JXErrors: Error {
     case cannotCreatePromise
```

Comment:
Grouped these a bit on cannotXXX and valueNotXXX cases. The generalization was needed in some of the new code, but I did also delete some more specific errors that you may or may not want me to add back - e.g. propertyAccessNonObject is just valueNotObject.